modules/validations/mutually_exclusive_tags.js

```diff
@@ -0,0 +1,46 @@
+import { t } from '../core/localizer';
+import { utilDisplayLabel } from '../util';
+import { validationIssue } from '../core/validation';
+
+export function validationMutuallyExclusiveTags(/* context */) {
+    const type = 'mutually_exclusive_tags';
+
+    // https://wiki.openstreetmap.org/wiki/Special:WhatLinksHere/Property:P44
+    const tagPairs = [
+        ['noname', 'name'],
```

Comment:
#6411 added a check for `name` matching `not:name`. It isn’t quite the same thing as checking whether two keys are merely present, but I wonder if it should be integrated into this validator rule somehow:

https://github.com/openstreetmap/iD/blob/bb17d5c24eb6c58724b8be83c646bfe0aa9cf3ad/modules/validations/suspicious_name.js#L154-L170

In that issue, we considered it important to add a field for `not:name` so that the user wouldn’t have to dig through the raw tags to resolve the issue. I don’t think any of the negative keys here have fields yet. Maybe it wouldn’t block this PR at all, but we should consider adding fields to id-tagging-schema separately.